style.css

```diff
@@ -0,0 +1,131 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;700&display=swap');
+
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  padding: 64px 150px;
+}
```

Comment:
- [ ] As you see on the screenshot, your mobile design doesn't fit the mobile screen because of the padding and the width you added to the body. Let's improve your design by changing the padding to `padding: 10px`, and changing the width to `width: 100%`.

<img width="203" alt="image" src="https://user-images.githubusercontent.com/77190203/159251359-2342f5de-d66c-48e9-9a51-cf52aa7ac604.png">